Nos/Views/Discover/FeaturedAuthor+Cohort5.swift

```diff
@@ -0,0 +1,271 @@
+            npub: "npub1fdyax3st9gcqj7zhqres3fw9pf7qrkukppv3fs7w2mcjcsqyrxjqcgcxcn",
+            categories: [.nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "David Hood",
+            npub: "npub12aew64q5lsxpcc98lskha0564gtjn30cw7vdueqnyj06xjqxtmksahdrlg",
+            categories: [.nzAustralia, .news]
+        ),
+        FeaturedAuthor(
+            name: "Kris Sowersby",
+            npub: "npub1t6n8u5sa80m6sx53m7uge6afktek4me982wvclr3xe5tfeyg34fslkxqcz",
+            categories: [.art, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "Greenpeace Atoearoa",
+            npub: "npub1mn78z9rssfq2yjqdwsa3h4qwu6zpw4xku9ppfldkde4ycwnpch4qhmcssl",
+            categories: [.activists, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "Bill Bennett",
+            npub: "npub1gpcxn339ap3r9tju2uy6d5k8rmud7f8kr579w7zv54qyevgz9xsqdtr4us",
+            categories: [.news, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "The Spinoff",
+            npub: "npub1qngl40pxy06k3hrqp4ltkn4p5yacen7u93du58v4tamf73tzaqhs6dfy8g",
+            categories: [.news, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "Dan Selvin",
+            npub: "npub16jjukmhnvflj92dv2jr8z6udfhzyyuyf3mckmf6afws9w4xdhhzs745kmv",
+            categories: [.news, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "Emma Cook",
+            npub: "npub18pl5kgxl47e5pssxl9q38m2v7ez477gw3nz220jz2vslnjreqq8s2yxl9m",
+            categories: [.art, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "Jack Yan",
+            npub: "npub179lhhjz30t3n4utkkn26qzgmpx44v9s5qdtwjs5686px8ulw7dfszxmfgp",
+            categories: [.news, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "Lucire",
+            npub: "npub1l4s4mtt96znwu3plfeyupk37y63xfapw5e7kjn7uavuw02lvav5q3ngc3s",
+            categories: [.news, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "Ian Griffin",
+            npub: "npub19p6edn6lvz9dmegm7s8vyz9w5w0ejzgztke8rcswxgmqr35h8feszt36jc",
+            categories: [.sciFi, .nzAustralia]
+        ),
+        FeaturedAuthor(
+            name: "JSM",
+            npub: "npub1uqeexjx2djkfwzxdnrnrrch5h2k4xn0uapcgsxm94ftaxrlhy5lqywjckg",
+            categories: [.tech]
+        ),
+        FeaturedAuthor(
+            name: "Carla",
+            npub: "npub1hu3hdctm5nkzd8gslnyedfr5ddz3z547jqcl5j88g4fame2jd08qh6h8nh",
+            categories: [.random]
+        ),
+        FeaturedAuthor(
+            name: "Wavlake",
+            npub: "npub1yfg0d955c2jrj2080ew7pa4xrtj7x7s7umt28wh0zurwmxgpyj9shwv6vg",
+            categories: [.music]
+        ),
+        FeaturedAuthor(
+            name: "Angela",
+            npub: "npub19vvkfwy9mcluhvehw7r56p4stsj5lmx4v9g3vgkwsm3arpgef8aqsrt562",
+            categories: [.random]
+        ),
+        FeaturedAuthor(
+            name: "J",
```

Comment:
```suggestion
            name: "Nela",
```

Reply:
There is actually a J on line 51: https://docs.google.com/spreadsheets/d/1tyeqgborx7dpTbV-fQI7KAK0nXISgpQ_qfRt2edoqRg/edit?gid=0#gid=0

Reply:
But I missed out Nela. Thanks for catching that.

Reply:
Oh wow -- there is! I assumed J had to be wrong so that's why I looked up the npub.